### ISSUE TITLE
Feature request: Have Estimator display Loss and Metrics for Every Epoch and not Every Step

### ISSUE THREAD

**Issue by selcouthlyBlue** [tensorflow/tensorflow#16506]
Most of the papers I’ve read measure the time it takes to train a model with every epoch and not every step. If it isn’t possible to display the loss only for every epoch, I think it would be nice to print when an epoch has passed.

**Comment by drpngx** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
